app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdUiRendererTest.kt

```diff
@@ -22,6 +22,7 @@ class ScanSimpleIdUiRendererTest {
     // then
     verify(ui).showSearchingForPatient()
     verify(ui).hideScanError()
+    verify(ui).setToolBarTitle(searchingModel.openedFrom)
```

Comment:
Can we pass the actual `OpenedFrom` value that we expect to have when asserting, please?